src/components/tutorial-collection-cards/components/card-badges/index.tsx

```diff
@@ -24,6 +24,7 @@ const PRODUCT_ICON_MAP: Record<ProductOption, JSX.Element> = {
 	vault: <ProductIcon productSlug="vault" />,
 	vagrant: <ProductIcon productSlug="vagrant" />,
 	waypoint: <ProductIcon productSlug="waypoint" />,
+	sentinel: <ProductIcon productSlug="sentinel" />,
```

Comment:
Sentinel does not have a product icon so this change is unnecessary

Reply:
```suggestion
```